packages/office-ui-fabric-react/src/components/ActivityItem/ActivityItem.Props.ts

```diff
@@ -31,34 +36,14 @@ export interface IActivityItemProps extends React.AllHTMLAttributes<HTMLElement>
   isCompact?: boolean;
 
   /**
-   * A renderer for the description of the current activity.
-   */
-  onRenderActivityDescription?: IRenderFunction<IActivityItemProps>;
-
-  /**
-   * A renderer that adds the text of a comment below the activity description.
-   */
-  onRenderComments?: IRenderFunction<IActivityItemProps>;
-
-  /**
-   * A renderer to create the icon next to the activity item.
-   */
-  onRenderIcon?: IRenderFunction<IActivityItemProps>;
-
-  /**
-   * A renderer adds a time stamp. If not included, timeStamp is shown as plain text below the activity.
-   */
-  onRenderTimeStamp?: IRenderFunction<IActivityItemProps>;
```

Comment:
This is a completely breaking change and violates semver. You can check this into the 5.0 branch if you need the changes. Or you can make it backwards compatible. But you need to realize that if any partner uses these now, you are creating build breaks in their code by adding breaking changes.